.ci/generate.jenkinsfile

```diff
@@ -352,15 +352,20 @@ node('Linux') {
                 withCredentials([usernamePassword(credentialsId: 'docker-credentials', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
                     sh 'docker login --username $USERNAME --password $PASSWORD'
                 }
+                String tagVersion = params.conan_version
+                String parsedVersion = parseVersion(params.conan_version)
+                if (parsedVersion[0] == 2) {
+                    tagVersion = "2.0.0-pre"
```

Comment:
why pre-? it's not pre-release, it's alpha/beta?

Reply:
for now, we only want to build the latest 2.0.0 version. It does not matter if it's alpha or beta, because we will need to update each version and validate the recipe compatibility. If we start to provide a single image for each 2.x alpha/beta release, we will publish and deprecate in few weeks.

Reply:
well, it might be harder to handle, as you can't compute image name siimply from `conan --version` any more, you need a specific branch for V2, but okay